.github/workflows/ort.yml

```diff
@@ -34,6 +34,11 @@ jobs:
         rm ort.tar.gz
         echo "ort-${{ env.ORT_VERSION }}/bin" >> $GITHUB_PATH
 
+    - name: Copy ORT Config
+      run: |
+        mkdir -p $HOME/.ort/config
+        cp ort-server/.github/ort.config.yml $HOME/.ort/config/config.yml
```

Comment:
Alternatively, commit `ort.config.yml` as `config.yml` to a directory that you can make the `ORT_CONFIG_DIR` environment variable point to.